libs/core/src/components/sage-input/sage-input.scss

```diff
@@ -0,0 +1,111 @@
+:host {
+  /**
+
+    * @prop --background: Background of the input
+    * @prop --background-disabled: Background color of a disabled input
+    *
+    * @prop --color: Color of the input text
+    * @prop --color-error: Color of the input text when error is present
+    *
+    * @prop --border-color-default: Border color of the input text
+    * @prop --color-hover: Border color of the input text when hovered
+    *
+    * @prop --label-font-size: Font size of the label text
+    * @prop --label-font-weight: Font weight of the label text
+    * @prop --label-line-height: Line height of the label text
+    * @prop --label-margin-bottom: Bottom margin of the label
+    *
+    * @prop --field-font-size: Font size of the field text
+    * @prop --field-font-weight: Font weight of the field text
+    * @prop --field-line-height: Line height of the field text
+    * @prop --field-padding-block: Block padding for the field text
+    * @prop --field-padding-inline: Inline padding for the field text
+    *
+    * @prop --hint-font-size: Font size of the hint text
+    * @prop --hint-font-weight: Font weight of the hint text
+    * @prop --hint-line-height: Line height of the hint text
+    * @prop --hint-margin-top: Bottom margin of the hint
+    */
+  --background: #fff;
+  --background-disabled: #f9fafa;
+
+  --color: #60666c;
+  --color-error: #cf3c32;
+
+  --border-color-default: #d3d5d9;
+  --border-color-hover: #b5bac0;
+
+  --focus-box-shadow: 0 0 0 4px #8ecafb;
+
+  --label-font-size: 16px;
+  --label-font-weight: 500;
+  --label-line-height: 24px;
+  --label-margin-bottom: 8px;
+
+  --field-font-size: 16px;
+  --field-font-weight: 400;
+  --field-line-height: 24px;
+  --field-padding-block: 8px;
+  --field-padding-inline: 16px;
+
+  --hint-font-size: 12px;
+  --hint-font-weight: 400;
+  --hint-line-height: 20px;
+  --hint-margin-top: 8px;
+
+
+  display: inline;
+}
+
+.sage-input {
```

Comment:
Non-blocking and more of a thought. How should we format our CSS styles? Does this work? Or would something like `:host(.sage-input)`? I've seen Ionic do this but I'm still not sure what would make more sense for our needs.

Reply:
I'll have to do some discovery about the best way to do this. This is great to bring up and something we'll need to have guidelines around as a team.